A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -2077,17 +2077,19 @@ var A320_Neo_UpperECAM;
                 line.style.strokeWidth = "4";
                 _svgRoot.appendChild(line);
                 this.valueText = A320_Neo_UpperECAM.createSVGText("--.-", "Value", this.getValueTextX(), "88%", "bottom");
-                this.valueText2 = A320_Neo_UpperECAM.createSVGText("", "decimal", this.getValueTextX2(), "88%", "bottom");
-                this.valueTextpoint = A320_Neo_UpperECAM.createSVGText("", "decimalpoint", this.getValueTextXpoint(), "88%", "bottom");
                 //createRectangle(_class, _x, _y, _width, _height) {
                 this.N2Box = A320_Neo_UpperECAM.createRectangle("activeEngine", this.getBoxX(), "70", "150", "40");
                 _svgRoot.appendChild(this.N2Box);
                 _svgRoot.appendChild(this.valueText);
-                _svgRoot.appendChild(this.valueText2);
-                _svgRoot.appendChild(this.valueTextpoint);
             }
             this.refresh(false, 0, 0, true);
         }
+
+        formatN2DecimalSvg(N2value, N2digits) {
```

Comment:
I'd suggest this should be just `formatDecimalSvg`, and the arguments `value` and `digits`. It can work for anything, and it's in a generic class (although looks like the other user (fuel flow) misses the call to this due to `_valueDisplayPrecision` being 0?

Reply:
@tracernz just made a few more changes for the throttle percent displayed on the top right. Had to create a different function for `Panelbase` as the scope of the previous one is limited to the `Engine`